ONSdigital/eq-author
# DIGEQ-80 UX Alignment

**What**
1. Adding UX wrapper to surveys list page and fixing navigation.
2. Updated layout to match designs.
3. Using the new foundation based styles this commit updates the survey list page and  the questionnaire detail / build view with initial styles.
4. Added new questionnaire builder template.
5. Display success message after questions saved
6. Linking in the new questionnaire builder and the old reviewed/publish functionality
7. Updating unit tests

**Pre-requisites**
1. A user that can access the digital eq system
2. A user which has previously added a survey and created at least one questionnaire

**How to test**
1 Check out this branch: git checkout digeq-80-ux-cleanup
2 Run the database migrations python manage.py migrate
3 Run the server: python manage.py runserver
4 Browse to http://127.0.0.1:8000/login/ and login with valid credentials
5 Browse to http://127.0.0.1:8000/surveys/
6. Select questionnaire
8. Click edit

**Who can review**
Anyone on Digital EQ apart from @warren-methods and @dhilton
